### PR TITLE
Unhardcoded and self-documented render sequences

### PR DESCRIPTION
**Pull request by Mailaender**
This exposes our default sequence name "idle" to https://github.com/OpenRA/OpenRA/wiki/Traits.

### CODE REVIEW

**Comment by Phrohdoh** [comment]
Exposing is not documenting.

File: OpenRA.Mods.RA/Traits/Render/RenderUnitReload.cs [hunk 1]
```diff
@@ -37,7 +41,7 @@ public RenderUnitReload(ActorInitializer init, RenderUnitReloadInfo info)
 
 		public override void Tick(Actor self)
 		{
-			var sequence = (armament.IsReloading ? "empty-" : "") + (attack.IsAttacking ? "aim" : "idle");
+			var sequence = (armament.IsReloading ? "empty-" : "") + (attack.IsAttacking ? info.AimSequence : info.DefaultSequence);
```
Comment (pchote) [inline OpenRA.Mods.RA/Traits/Render/RenderUnitReload.cs]: Don't forget "empty-" &rarr; info.EmptyPrefix.

Comment (Mailaender) [inline OpenRA.Mods.RA/Traits/Render/RenderUnitReload.cs]: Updated.

**Comment by pchote** [comment]
:+1: Code looks fine.  I haven't tried this ingame, but there's not much to go wrong here.

**Comment by penev92** [comment]
:+1:

**Comment by reaperrr** [comment]
Encountered no regressions on any of the shellmaps, can be merged after release is out :+1:

File: OpenRA.Mods.Common/Traits/Render/RenderSimple.cs [hunk 2]
```diff
@@ -18,6 +18,8 @@ namespace OpenRA.Mods.Common.Traits
 {
 	public class RenderSimpleInfo : RenderSpritesInfo, IRenderActorPreviewSpritesInfo, IQuantizeBodyOrientationInfo, ILegacyEditorRenderInfo, Requires<IBodyOrientationInfo>
 	{
+		public readonly string DefaultSequence = "idle";
```
Comment (pchote) [inline OpenRA.Mods.Common/Traits/Render/RenderSimple.cs]: The convention we have set in our other render traits (mainly With*Overlay) is to call this just Sequence.  The "Default" part doesn't add anything useful.

Comment (Mailaender) [inline OpenRA.Mods.Common/Traits/Render/RenderSimple.cs]: Renamed.

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/1193be43317e37e82b4be5519b876da959aa3ad3%5E...1193be43317e37e82b4be5519b876da959aa3ad3

**Pull request merged**